engine/src/main/java/org/terasology/logic/players/FirstPersonClientSystem.java

```diff
@@ -37,11 +36,14 @@
 import org.terasology.math.geom.Quat4f;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.network.ClientComponent;
+import org.terasology.network.NetworkComponent;
+import org.terasology.physics.components.RigidBodyComponent;
 import org.terasology.registry.In;
 import org.terasology.rendering.world.WorldRenderer;
 
 @RegisterSystem(RegisterMode.CLIENT)
 public class FirstPersonClientSystem extends BaseComponentSystem implements UpdateSubscriberSystem {
+
     private static final int USEANIMATIONLENGTH = 200;
```

Comment:
Slightly unrelated: Don't we use a naming scheme with underscores for static members, i.e., `USE_ANIMATION_LENGTH` or `USE_ANIMATION_DURATION`? Easy change alongside other changes in this file.